deployment/services/proxy.ts

```diff
@@ -89,6 +89,10 @@ export function deployProxy({
         service: graphql.service,
         requestTimeout: '60s',
         retriable: true,
+        rateLimit: {
+          maxRequests: 7,
```

Comment:
That’s super low, no? The session is validated on every page visit (the initial one) and there’s a chance we get a oauth callback with a redirect to a new page that increases the amount by a lot

Reply:
The calls for `/auth-api` made only when you first load the page (and then it's SPA). It's 3 calls, so you are right, it needs adjustments. 
How about max 10 requests a minute?

Reply:
@kamilkisiela we can also set `burst: 3` and it will ignore the specific burst config from the token validation

Reply:
Let's test it on staging and see if none of the auth flows are affected

Reply:
@kamilkisiela seems like `maxRequests: 10` with burst works well (even when I'm refreshing very quickly). Also tried to login/logout multiple times.
When trying to send requests from curl, it blocks me at some point.